portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,71 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio Page</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kaleab Gebremichael</h1>
+
+      <div class="navbar">
+        <a href="#about">About me</a>
+        <a href="#work">Work Experience</a>
+        <a href="#hobbies">Hobbies</a>
+        <a href="#contact">Contact Me</a>
+      </div>
+
+      <p id="about"> My name is Kaleab Gebremichael and I'm a rising junior at Vanderbilt. I am majoring in Computer Science and Mathematics.</p>
+      <img class="photo" src="Images/naruto-1.png">
+      
+      <hr>
+
+      <h3 id="work">Work Experience</h3>
+      <table>
+          <thead>
+              <tr>
+                <th>Dates</th>
+                <th>Work</th>
+              </tr>
+          </thead>
+          <tbody>
+            <tr>
+                <td>2019 Summer</td>
+                <td>Microsoft TNT</td>
+            </tr>
+            <tr>
+                <td>2019-Present</td>
+                <td>Undergrad CS TA</td>
+            </tr>
+            <tr>
+                <td>2018-Present</td>
+                <td>Vandy Apps Secretary</td>
+            </tr>
+          </tbody>
+      </table>
+
+      <hr>
+
+      <h3 id="hobbies"> Hobbies </h3>
+
+      <ul> 
+          <li>Playing Ping Pong</li>
+          <li>Swing Trading</li>
+          <li>Coding</li>
+      </ul>
+
+      <hr>
+
+
+      <h3 id="contact"> Contact Me </h3>
+      <a target="_blank" href="https://github.com/Kaleab470">Github</a>
+      <a target="_blank" href="https://linkedin.com/in/kaleab-gebremichael">LinkedIn</a>
+      <a target="_blank" href="mailto:kaleabsolomon470@gmail.com">Email</a>
+
+
```

Comment:
Get rid of some of these extra new lines.
1 blank line is usually good if you want to show some kind of break in the logic. >1 is almost never really used